crates/api_common/src/utils.rs

```diff
@@ -1000,6 +1008,56 @@ pub fn send_webmention(post: Post, community: &Community) {
   };
 }
 
+pub fn community_follower_state(community: &Community) -> CommunityFollowerState {
+  if community.local {
+    // Local follow is accepted immediately
+    CommunityFollowerState::Accepted
+  } else if community.visibility == CommunityVisibility::Private {
+    // Private communities require manual approval
```

Comment:
Shouldn't the private check go first? Otherwise local private ones immediately get accepted. I forget how federation works with these.

Reply:
True